ui/src/app/app-routing.module.ts

```diff
@@ -111,10 +110,14 @@ const routes: Routes = [
               {
                 /* TODO The children under ./views need refactoring to use the data
                  * provided by the route rather than double-requesting it.
+                 *
+                 * TODO angular2react: ideally, we should be able to just render the AppComponent here and let
```

Comment:
One idea in the interim could be to let the signed in component own `/**`, and map once to AppRouting. I'm not sure whether this will continue to cause issues with angular router interactions.

Reply:
I did try that actually and spent quite a bit of time trying to get it to work but the React `BrowserRouter` wouldn't update what it should render even though the URL would change.. Some of the things I tried were 
  - add the `withRouter` prop to all components
  - wrap in `<Switch>`
  - change the `NavStore.navigate*` functions to change routes by pushing to the History API which react-router uses as its backend